### ISSUE TITLE
add jsonmove "used time".

### ISSUE THREAD

**Issue by o-jill** [o-jill/washcrus#23]
it is normal that used time for each move is written in a kifu.

to get used time, you have to calc time difference btw 2 moves.

date time at last move is in matchinfofile and you can get current date-time.
you can simply subtract them to calc the interval.

**Comment by o-jill** [comment]
zero information was added in 5f66d9f.

**Comment by o-jill** [comment]
date-time infomation can be recognized with Time.parse('yyyy/mm/dd hh:mm:ss').
and time difference can be calc with float diff [sec] = Time1 - Time2.

**Comment by o-jill** [comment]
anyway implemented in 9a0fa7f712.
but not confirmed yet at all!!

**Comment by o-jill** [comment]
jkf gem does not support jkf format fully.
```ruby
mandatory_in_jkf_gem = {
 'now': {
  'm' => 0, 's' => 0
 },
 'total': {
  'h'=> 0, 'm' => 0, 's' => 0
 }
}
```
but in original jkf, 'h' is not mandatory.
so, it might be better to fix this problem by myself by fork-ing.

**Comment by o-jill** [comment]
implemented up to 3c36fcc.
